xs/src/libslic3r/EdgeGrid.cpp

```diff
@@ -773,11 +773,11 @@ void EdgeGrid::Grid::calculate_sdf()
 				// For each corner of this cell and its 1 ring neighbours:
 				for (int corner_y = -1; corner_y < 3; ++ corner_y) {
 					coord_t corner_r = r + corner_y;
-					if (corner_r < 0 || corner_r >= nrows)
```

Comment:
I wouldn't recommend just blindly casting signed integers to unsigned integers - if anything, I would cast `nrows` to a signed integer.